package.json

```diff
@@ -4,7 +4,7 @@
   "main": "dist/webexSDKComponentAdapter.cjs.js",
   "module": "dist/webexSDKComponentAdapter.esm.js",
   "scripts": {
-    "build": "rollup -c",
+    "build": "rollup -c --environment INCLUDE_DEPS,BUILD:production",
```

Comment:
Can you explain the environment variables you just added and what benefits they have?

Reply:
don't feel like it hehe. its removed anyways.